resources/can/fh16mapping.can

```diff
@@ -9,54 +9,66 @@ using ODVDFH16Truck;
+    driveaxle1load at bit 16 for 16 bit is unsigned little endian multiply by 0.5 add 0 with range [0, 32767.5];
+    frontaxle1load at bit 0 for 16 bit is unsigned little endian multiply by 0.5 add 0 with range [0, 32767.5];
+}
 // CAN message for reading the vehicle dynamics.
-vehicledynamics.yawrate in 0x104 at bit 32 for 16 bit is unsigned little endian multiply by 0.000122 add -3.92 with range [-3.92, 4.07527];
-vehicledynamics.acceleration_y in 0x104 at bit 16 for 16 bit is unsigned little endian multiply by 0.00048828125 add -15.687 with range [-15.687, 16.31251171875];
-vehicledynamics.acceleration_x in 0x104 at bit 0 for 16 bit is unsigned little endian multiply by 0.00048828125 add -15.687 with range [-15.687, 16.31251171875];
-
+CAN Message 0x104 vehicledynamics : 6 {
+    yawrate at bit 32 for 16 bit is unsigned little endian multiply by 0.000122 add -3.92 with range [-3.92, 4.07527];
+    acceleration_y at bit 16 for 16 bit is unsigned little endian multiply by 0.00048828125 add -15.687 with range [-15.687, 16.31251171875];
+    acceleration_x at bit 0 for 16 bit is unsigned little endian multiply by 0.00048828125 add -15.687 with range [-15.687, 16.31251171875];
+}
 // CAN message for reading the vehicle speed.
-vehiclespeed.vehiclespeedpropshaft in 0x103 at bit 0 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-
+CAN Message 0x103 vehiclespeed : 2 {
+    vehiclespeedpropshaft at bit 0 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
+}
 // CAN message for reading the wheel speed.
-wheelspeeds2.driveaxle2wheelspeedright in 0x102 at bit 16 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-wheelspeeds2.driveaxle2wheelspeedleft in 0x102 at bit 0 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-
-wheelspeeds1.driveaxle1wheelspeedright in 0x101 at bit 48 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-wheelspeeds1.driveaxle1wheelspeedleft in 0x101 at bit 32 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-wheelspeeds1.frontaxle1wheelspeedright in 0x101 at bit 16 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-wheelspeeds1.frontaxle1wheelspeedleft in 0x101 at bit 0 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
-
+CAN Message 0x102 wheelspeeds2 : 4 {
+    driveaxle2wheelspeedright at bit 16 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
+    driveaxle2wheelspeedleft at bit 0 for 16 bit is unsigned little endian multiply by 0.00390625 add 0 with range [0, 255.99609375];
+}
+CAN Message 0x101 wheelspeeds1 : 8 {
```

Comment:
@fgiaimo Could you improve the language design slightly:

CAN Message 0x101 wheelspeeds1 : 8 {
-->
CANMessage 0x0101 wheelspeeds1 is 8 bytes {